### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by huu4ontocord**

### CODE REVIEW

**Review by omri374 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # piisa
-Personal information identification standard
+This is the Personal information identification standard (Piisa) repository. We are working on an on open specifications for interoperating between variouos privacy API frameworks.
```
Comment (paulovn) [inline README.md]: there is a typo here: _we are working on an **on** open_

**Review by paulovn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (omri374) [reply README.md]: good catch

Comment (omri374) [inline README.md]:
```suggestion
This is the Personal information identification standard (Piisa) repository. We are working on an open specification for interoperating between various privacy API frameworks. 
```

Comment (omri374) [reply README.md]: Fixed

**Comment by omri374** [comment]
@paulovn @ontocord please review, I've added some content to the README.md

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+PIISA stands for a *Personal Identifiable Information Standard*. This standard allows seamless interoperability between various PII detection frameworks.
+
+## Specification
+
+[Click here for the latest specification document](docs/specs.md).
+
+## Who are we
+
+We are a team of privacy enthusiasts who are interested in improving de-identification across multiple domains.
```
Comment (paulovn) [inline README.md]: maybe something in the lines of _PII management_ would be more encompassing than "de-identification" (which is a key application, granted, but need not be the only one)

Or _PII management, including de-identification_, which has it all :-)

**Review by paulovn — DISMISSED** [review]
Great!

File: README.md (hunk 2 above)
Comment (omri374) [reply README.md]: Good idea. I'll change it

Comment (omri374) [reply README.md]: Changed. Could you please re-review and re-approve?

**Review by paulovn — APPROVED** [review]
ok!

**Pull request merged**